docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1180,6 +1180,92 @@ privacy-public-key-file="Orion/nodeKey.pub"
 
 Path to the [public key of the Orion node](../../Concepts/Privacy/Privacy-Overview.md#besu-and-orion-keys).     
 
+### privacy-tls-enabled
+
+```bash tab="Syntax"
+--privacy-tls-enabled[=<true|false>]
+```
+
+```bash tab="Command Line"
+--privacy-tls-enabled=false
+```
+
+```bash tab="Environment Variable"
+BESU_PRIVACY_TLS_ENABLED=false
+```
+
+```bash tab="Configuration File"
+privacy-tls-enabled=false
+```
+
+Enables [TLS with the Private Transaction Manager](../../Concepts/Privacy/Privacy-Overview.md#private-transaction-manager).
```

Comment:
TLS with the -> TLS on communication with the?

Reply:
Done